enclave_build/docs/cache.md

```diff
@@ -0,0 +1,127 @@
+# Image Cache
+
+When building an enclave image using the Docker daemon, the image will be pulled and stored in the Docker cache. We want to implement the same functionality when choosing to build an OCI image. The resulted cache will follow the [OCI specification](https://github.com/opencontainers/image-spec), allowing tools like `linuxkit` to use it as its own cache, and also adding images from OCI archive with the same format.  
```

Comment:
Please limit the lines in md files to 80 chars; this is the general recommendation, otherwise it is difficult to read such files in an IDE for example.

Reply:
I think rust-fmt is 100 or 120 chars/line and I think it's fine if we keep a limit that works on a modern PC. But the line needs to be split for sure.

Reply:
Yes, I don't have a strong preference for 80, mostly just for splitting the lines.